smartautoclicker/src/main/res/layout/dialog_permissions.xml

```diff
@@ -15,6 +15,11 @@
  * You should have received a copy of the GNU General Public License
  * along with this program; If not, see <http://www.gnu.org/licenses/>.
 -->
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
```

Comment:
You need to correct the indent on the elements inside your ScrollView element.
Simply add 4 spaces to the LinearLayout and its content

Reply:
I have changed the indent in this file. :)